### PR TITLE
SLI-1235 Do not use unsafe VFS recursion

### PR DESCRIPTION
**Pull request by damien-urruty-sonarsource**

### CODE REVIEW

File: src/main/java/org/sonarlint/intellij/fs/VirtualFileSystemListener.kt [hunk 1]
```diff
@@ -46,7 +48,7 @@ import org.sonarsource.sonarlint.plugin.api.module.file.ModuleFileEvent
  * The BulkFileListener is not tied to a specific project but global to the IDE instance
  */
 class VirtualFileSystemListener(
-    private val fileEventsNotifier: ModuleFileEventsNotifier = ModuleFileEventsNotifier()
+    private val fileEventsNotifier: ModuleFileEventsNotifier = ModuleFileEventsNotifier(),
```
Comment (nquinquenel) [inline src/main/java/org/sonarlint/intellij/fs/VirtualFileSystemListener.kt:51]: Is it from the sonar code style?

**Review by nquinquenel — APPROVED** [review]
LGTM

File: src/main/java/org/sonarlint/intellij/fs/VirtualFileSystemListener.kt (hunk 1 above)
Comment (damien-urruty-sonarsource) [reply src/main/java/org/sonarlint/intellij/fs/VirtualFileSystemListener.kt:51]: Yes
image

Comment (nquinquenel) [reply src/main/java/org/sonarlint/intellij/fs/VirtualFileSystemListener.kt:51]: I have the same but it does not add this comma 🤔

**Pull request merged**